spring-boot-project/spring-boot-docs/src/main/asciidoc/production-ready-features.adoc

```diff
@@ -1644,7 +1644,7 @@ Spring Boot registers the following core metrics when applicable:
 ** Number of classes loaded/unloaded
 * CPU metrics
 * File descriptor metrics
-* Kafka consumer metrics
+* Kafka consumer metrics (`spring.jmx.enabled` must be set to `true`)
```

Comment:
This is a good idea. I'd rather change the note to something like "JMX support should be enabled" or something like that with a link to the section that explains how to do it rather than the reference to the property.

Reply:
@snicoll Thanks for the feedback! I changed it as suggested.

Reply:
Thanks for the quick feedback. I meant that the text on "JMX support" should be a link to the relevant section. Have you missed it or do you disagree with that approach?

Reply:
@snicoll Sorry. I overlooked your comment. I changed it again. Please let me know if it isn't what you meant.